package.json

```diff
@@ -39,6 +39,7 @@
     "@babel/eslint-parser": "^7.18.2",
     "@babel/node": "^7.13.0",
     "@babel/plugin-proposal-class-properties": "^7.13.0",
+    "@babel/plugin-transform-async-generator-functions": "^7.22.3",
```

Comment:
Missing package lock for this dependency

Reply:
Added